### DISCUSSION TITLE
Does Isar support native Dart?

### DISCUSSION THREAD

**Discussion by buraktabn** [isar/isar#515] [Help]
I'm working on a CLI project written in Dart. I wonder If I can use Isar with native Dart to run on Ubuntu.

**Comment by guyluz11** [comment]
I have tested it on Dart native with Kubuntu and it is working.

Don't forget at the start of your program to initialize it (not needed in Flutter because of [`isar_flutter_libs`](https://pub.dev/packages/isar_flutter_libs) package)

```dart
await Isar.initializeIsarCore(download: true)
```
